pkg/webui/lib/components/with-locale.js

```diff
@@ -67,23 +71,29 @@ export default class UserLocale extends React.PureComponent {
   }
 
   check(prev, props) {
-    const current = (prev.user && prev.user.language) || prev.env.config.language || defaultLanguage
+    const current = (prev.user && prev.user.language) || prev.env.config.language
```

Comment:
Since there is no way to change the locale atm, I think the comparison of previous and current props is redundant. Maybe we should just go with `prev.env.config.language` here for now, which is always present.

Reply:
> Since there is no way to change the locale atm

But there will be an option later. Lets keep changes to a minimum.